functions/form-payment-reconciliation/index.js

```diff
@@ -0,0 +1,149 @@
+const { SSMClient, GetParameterCommand } = require('@aws-sdk/client-ssm')
+const { generateLinkToCloudwatch, postToSlack } = require('./src/utils')
+
+const AWS_REGION = 'ap-southeast-1'
+
+const PARAMETER_STORE_NAME = 'staging-cron-payment'
+
+const SUB_DOMAIN = process.env.ENV_SITE_SUBDOMAIN || 'staging.'
+
+/** TODO: migrate to separate file */
+function getSecrets() {
+  const awsSsmClient = new SSMClient({ region: AWS_REGION })
+  const input = {
+    Name: PARAMETER_STORE_NAME,
+    WithDecryption: true,
+  }
+  const command = new GetParameterCommand(input)
+  return awsSsmClient
+    .send(command)
+    .then((d) => d.Parameter.Value)
+    .then((d) => {
+      const dn = d.split('\n')
+      return dn.reduce((accum, line) => {
+        const [key, value] = line.split('=')
+        accum[key] = value
+        return accum
+      }, {})
+    })
+}
+
+const getPendingQueries = (apiSecret) => {
+  return fetch(
+    `https://${SUB_DOMAIN}form.gov.sg/api/v3/payments/pendingPayments`,
+    {
+      headers: {
+        'x-cron-payment-secret': apiSecret,
```

Comment:
/nit: we have a convention of making custom formsg headers' prefix be `x-formsg-` (see [mail constants](https://github.com/opengovsg/FormSG/blob/develop/src/app/services/mail/mail.constants.ts#L11-L13) and [webhook service](https://github.com/opengovsg/FormSG/blob/develop/src/app/modules/webhook/webhook.service.ts#L151) for reference)

Reply:
fixed in [95ab1e8](https://github.com/opengovsg/FormSG/pull/6102/commits/95ab1e89b5031f60b29d3019a75651cfb24f589d)